401-installation-state.md

```diff
@@ -0,0 +1,445 @@
+
+The CNAB installation is defined as a JSON document. The specification does not require that installations are formatted as Canonical JSON.
+
+```json
+{
+  "bundleRepository": "hub.example.com/technosophos/hellohelm",
+  "bundleVersion": "0.1.2",
+  "created": "2018-08-30T20:39:55.549002887-06:00",
+  "custom": {},
+  "labels": {
+    "env": "dev",
+    "owner": "javier",
+    "cnab.io/app": "hello-world",
+    "cnab.io/appVersion": "v1.0.0"
+  },
+  "name": "hello-world",
+  "namespace": "demo"  
+}
+```
+Source: [401.01-installation.json](examples/401.01-installation.json)
+
+The fields above are defined as follows:
+
+- `bundleRepository` (REQUIRED): The repository of the associated bundle.
+- `bundleVersion` (REQUIRED): The version of the associated bundle.
+- `created` (REQUIRED): A [timestamp](#timestamps) indicating when this _installation_ was created.
+- `custom` (OPTIONAL): A section for custom extension data applicable to a given runtime.
+- `labels` (OPTIONAL): [Labels](#labels) are a set of key/value pairs of type string that MAY be used for querying. Labels defined on the bundle SHOULD be copied to the labels field on the installation. MUST follow the [CNAB Label Format]. 
+- `name` (REQUIRED): The name of the _installation_. Elsewhere, this field is referenced as the _installation name_. The format of this field must follow the same format used for the `installation` field in the [bundle.json file specification](101-bundle-json.md#the-bundlejson-file).
+- `namespace` (OPTIONAL): The [namespace](#namespaces) of the installation. MUST follow the [CNAB Namespace Format].
+
+
+### Claim
+
+A _claim_ is a record of the inputs to an action against a CNAB _installation_.
+
+The word _claim_ was chosen because it represents the relationship between a certain CNAB host environment and the resources that were created by a CNAB runtime in that host environment. In this sense, an environment takes responsibility for those resources if and only if it can _claim_ them. A claim is an _external assertion of ownership_. That is, the claim itself is not "installed" into the host environment. It is stored separately, possibly in an entirely different location.
+
+An installation MAY change over time, as a particular bundle is installed, then upgraded. In CNAB, each time a modifying operation is performed (Such as `install`, `upgrade`, or custom operations that are not read-only), the installation gets a new _revision_. A revision is a unique identifier that identifies the combination of an _installation_ and a modification of that installation. Revision is stored on the claim. For example, when a CNAB bundle is installed, the installation will have an initial revision ID. When that installation is upgraded, it will have a new revision ID. How revisions are used is outside of the scope of this document, but it is safe to assume that if a revision ID has changed, one or more artifacts owned by the installation has also been changed.
```

Comment:
```suggestion
An installation MAY change over time, as a particular bundle is installed, then upgraded. In CNAB, each time a modifying operation is performed (such as `install`, `upgrade`, or custom operations that are not read-only), the installation gets a new _revision_. A revision is a unique identifier that identifies the combination of an _installation_ and a modification of that installation. Revision is stored on the claim. For example, when a CNAB bundle is installed, the installation will have an initial revision ID. When that installation is upgraded, it will have a new revision ID. How revisions are used is outside of the scope of this document, but it is safe to assume that if a revision ID has changed, one or more artifacts owned by the installation has also been changed.
```